### DISCUSSION TITLE
[Question]: Start with existing OpenAI Assistant thread

### DISCUSSION THREAD

**Discussion by rrdd** [danny-avila/LibreChat#3782] [Q&A]
### What is your question?

Is it possible to open a thread from an existing open ai assistant conversation?
Similar to https://platform.openai.com/playground/assistants?assistant=asst_1234567789...&thread=thread_1234567890

### More Details

I would like to start or open an existing openai assistant conversation referenced by the assistant id and thread.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
at the moment, no. In theory it's possible
